docs/Troubleshooting.md

```diff
@@ -9,6 +9,20 @@ next: mock-functions
 
 Uh oh, something went wrong? Use this guide to resolve issues with Jest.
 
+### Tests are Failing and You Don't Know Why
+
+Try using the debugger built into >= Node 6.3. 
+
+Place a `debugger;` statement in any of your tests, and then, in your project's directory, run:
+
+`node --debug-brk --inspect ./node_modules/.bin/jest --i [any other arguments here]`
```

Comment:
This problem listed here https://github.com/facebook/jest/issues/1652

It will work if you use `node debug --debug-brk  ./node_modules/.bin/jest -i` command